percy/environment.py

```diff
@@ -251,7 +251,7 @@ def commit_sha(self):
 
     @property
     def parallel_nonce(self):
-        return os.getenv('CIRCLE_WORKFLOW_WORKSPACE_ID') or os.getenv('CIRCLE_BUILD_NUM')
+        return os.getenv('CIRCLE_BUILD_NUM')
```

Comment:
What exactly does this number represent? I'm finding "The number of the CircleCI build."  as per https://circleci.com/docs/2.0/env-vars/ a bit vague. If a jobs gets parallelized into a set of separate sub-jobs, will CIRCLE_BUILD_NUM be different for each sub-job?

Reply:
I tested this and it seems to be the same for subjob, but does generate a new number for each rebuild.  TBH the documentation wasn't super helpful, so I resorted to rebuilding and looking at the env variables 

This is not true for WORKSPACE_ID (it does not generate a new number for each rebuild)